cfme/tests/automate/test_instance.py

```diff
@@ -280,3 +281,78 @@ def test_check_system_request_calls_depr_conf_mgmt(appliance, copy_instance):
         request=copy_instance.name
     )
     result.validate_logs()
+
+
+@pytest.fixture(scope="module")
+def copy_quota_instance(domain):
+    """Copy the default instance 'quota' to custom domain"""
+    miq = domain.appliance.collections.domains.instantiate("ManageIQ")
+
+    original_instance = (
+        miq.namespaces.instantiate("System")
+        .namespaces.instantiate("CommonMethods")
+        .classes.instantiate("QuotaStateMachine")
+        .instances.instantiate("quota")
+    )
+    original_instance.copy_to(domain=domain)
+
+    instance = (
+        domain.namespaces.instantiate("System")
+        .namespaces.instantiate("CommonMethods")
+        .classes.instantiate("QuotaStateMachine")
+        .instances.instantiate("quota")
+    )
+    return instance
```

Comment:
**Required **@ganeshhubale I think it is worth adding a finalizer here to clean up this domain, we want to leave the system the same as when the test started. Do you agree here

Reply:
@jawatts 
Actually, ```domain``` is global fixture(which creates custom domain) and we are copying default instance to this domain and domain`s teardown is already present there.
Once we delete the domain then all the things under it got deleted.

Reply:
@ganeshhubale Awesome, thanks!